frontend/src/component/project/Project/ProjectInsights/ProjectInsights.tsx

```diff
@@ -16,7 +17,11 @@ const Grid = styled(Box)(({ theme }) => ({
     gridTemplateColumns: 'repeat(10, 1fr)',
 }));
 
-const FullWidthContainer = styled(Container)(() => ({
+const NoBackgroundFullWidthContainer = styled(Box)(() => ({
+    gridColumn: '1 / -1',
+}));
+
+const FullWidthContainer = styled(Box)(() => ({
```

Comment:
There is only one full width container that is needed and used.

Reply:
Alright, I thought they were created for different purposes since it had a background. Will revert to one container without background then.